### PR TITLE
chore: almost all links fixed. only a few I can't find

### PR DESCRIPTION
**Pull request by ashley-o0o**
Fixed more links
Closes https://github.com/patternfly/patternfly-org/issues/3437

### CODE REVIEW

**Comment by ashley-o0o** [comment]
**What:** Closes [#3437](https://github.com/patternfly/patternfly-org/issues/3437)

File: packages/v4/patternfly-docs/content/design-guidelines/components/modal/modal.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Never use a passive dialog for non-critical information. Instead, use an [inline
 
 ### Input dialogs
 
-Users may need to provide additional input to complete an action. To accomplish this, use forms or other elements within the modal, like in a [modal wizard](/components/modal/react#with-wizard).
+Users may need to provide additional input to complete an action. To accomplish this, use forms or other elements within the modal, like in a [modal wizard](/components/modal).
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/design-guidelines/components/modal/modal.md:97]: I think the URL here would be better off as `/components/wizard#within-modal`

File: packages/v4/patternfly-docs/content/design-guidelines/components/switch/switch.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ All switch types display their state (on or off) through different methods and l
 
 **General guidelines**
 
-* Use a basic switch by default unless space is limited, then use a [switch without a label](/components/switch/react#without-label).
+* Use a basic switch by default unless space is limited, then use a [switch without a label](/components/switch).
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/design-guidelines/components/switch/switch.md:21]: this url should be `/components/switch#without-label`

File: packages/v4/patternfly-docs/content/training/html-css/fundamentals.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Add the badge inside the chip. To do this, add this block of code between `pf-v5
 7\n</span>`}
 </CopyCodeBlock>
 
-**Note:** Remember, this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip, it receives styling that gives it a margin. You can see that the badge in a chip has its own CSS variable, `--pf-v5-c-chip__c-badge--MarginLeft`, defined for the right margin under the <a href="/components/chip/html#css-variables">CSS Variables section of the chip documentation</a>.
+**Note:** Remember, this is a component and not a demo because the chip component includes styles that handle how the badge looks within the chip. For example, when the badge component is added to the chip, it receives styling that gives it a margin. You can see that the badge in a chip has its own CSS variable, `--pf-v5-c-chip__c-badge--MarginLeft`, defined for the right margin under the <a href="/components/chip/html">CSS Variables section of the chip documentation</a>.
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/training/html-css/fundamentals.md:70]: I think this link should have be working again now. I think you can revert this change.

File: packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ In the `style.css` file, in the `.pf-v5-c-alert{}` block, write the custom prope
 In reference to the formula described in Part 1, this should be: `--pf-v5-c-alert`
 
 #### Step 2.2 
-Add the modifier to the custom property name. As displayed in the CSS variables of PatternFly's <a href="/components/alert/html#css-variables" target="_blank">alert component documentation</a>, the success variation modifier class `pf-m-success` applies to `pf-v5-c-alert`. Add that modifier to the custom property.
+Add the modifier to the custom property name. As displayed in the CSS variables of PatternFly's <a href="/components/alert/html" target="_blank">alert component documentation</a>, the success variation modifier class `pf-m-success` applies to `pf-v5-c-alert`. Add that modifier to the custom property.
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/training/html-css/css-variables-and-overrides.md:45]: I think this link should have be working again now. I think you can revert this change.

File: packages/v4/patternfly-docs/content/training/html-css/fundamentals.md [hunk 5]
```diff
@@ -156,7 +156,7 @@ Adding the `pf-m-gutter` modifier class to the grid layout ensures there is equa
 
 Add the class `pf-m-gutter` after `pf-v5-l-grid` to the outermost wrapper, inside the quotation marks.
 It should look like: `<div class="pf-v5-l-grid pf-m-gutter">`
-**Note:** Learn how modifier classes work with layout classes by looking at PatternFly's <a href="/layouts/grid/html#usage" target="_blank">grid documentation</a>.
+**Note:** Learn how modifier classes work with layout classes by looking at PatternFly's <a href="/layouts/grid/html" target="_blank">grid documentation</a>.
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/training/html-css/fundamentals.md:159]: hm.... i think we want to keep the `#usage` in this url. It's not working at the moment because this URL was missing the `/v4/` in the href - but Katie is actively working to remove that from all URLS, so these links will fix themselves very soon I think. 

This change can be reverted.

**Review by nicolethoen — CHANGES_REQUESTED** [review]

**Review by nicolethoen — APPROVED** [review]
